kairon/shared/analytics/analytics_pipeline_processor.py

```
@@ -0,0 +1,163 @@
+
+        pipeline.delete()
+        return True
+
+
+    @staticmethod
+    def update_scheduled_task(event_id: str, bot: str, user: str, config: Dict):
+        """
+        Update a scheduled analytics pipeline event.
+        """
+        if not config.get("scheduler_config"):
+            raise AppException("scheduler_config is required!")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)